crates/milli/src/documents/mod.rs

```diff
@@ -220,20 +246,6 @@ mod test {
         assert!(documents.next_document().unwrap().is_none());
     }
 
-    #[test]
-    fn test_nested() {
```

Comment:
why did we remove this test?

Reply:
It was only testing the `EnhancedDocuments...` (?) that we no longer use in the new indexer.

Reply:
OK!